src/app/page/userProfile/profile.component.html

```diff
@@ -240,7 +233,7 @@ <h5 *ngIf="userProfile" class="title">You are signed in as {{userProfile.email}}
             <div class="col-sm-3">
             </div>
             <div class="col-sm-6">
-              <div *ngIf="currentPasswordNotMatch" class="alert alert-danger">
+              <div *ngIf="currentPasswordMessage==false" class="alert alert-danger">
```

Comment:
The condition should written as 

!currentPasswordMessage

Reply:
with !currentPasswordMessage condition my bad message will show on init before I click any button because !null will false and it isn't what I want

Reply:
Oh, javaScript is a kind of magic.